AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -108,8 +111,12 @@ def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size
         solidity = prop_axon.solidity
         # Eccentricity
         eccentricity = prop_axon.eccentricity
-        # Axon equivalent diameter in micrometers
-        axon_diam = prop_axon.equivalent_diameter * pixelsize
+        if circle_approx:
+            # Axon equivalent diameter in micrometers
+            axon_diam = prop_axon.equivalent_diameter * pixelsize
+        else: 
+            # Axon diameter using ellipse minor axis in micrometers
+            axon_diam = prop_axon.minor_axis_length * pixelsize
```

Comment:
So for the ellipse case, are we not storing the major and minor axes and providing it to the user? That's what I was expected, is that what we wanted?

Reply:
If I understand correctly, the current implementation only saves the minor axes as the equivalent diameter for the ellipse case, and the major axes is never used or saved, correct? Is this the behaviour that you wanted @mariehbourget ?

Reply:
As discussed in the meeting, we are only interested in the minor axis of ellipse for computing morphometrics, so we needn't worry about the major axis.